espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Http/Router.java

```diff
@@ -76,7 +77,7 @@ AppiumResponse route(IHTTPSession session) {
         System.out.println("Received " + method + " request for '" + uri + "'");
 
         if (!routerMap.containsKey(method)) {
-            routerMap.put(method, new HashMap<String, RequestHandler>());
+            routerMap.put(method, new ConcurrentHashMap<String, RequestHandler>());
```

Comment:
I think in this case the generics are required because we're putting a HashMap into a Map as a value. When I took the generics out it caused an error.